src/main/java/model/Offer.java

```diff
@@ -5,7 +5,7 @@
  */
 public class Offer {
 
-    private long id;
+    private Long id;
```

Comment:
Any option here would be good. So would you approve Long type? Approve is required to merge branch to master.